modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -628,104 +684,94 @@ class CV_EXPORTS_W VideoCapture
-    -   cv::CAP_PROP_EXPOSURE @copydoc cv::CAP_PROP_EXPOSURE
-    -   cv::CAP_PROP_CONVERT_RGB @copydoc cv::CAP_PROP_CONVERT_RGB
-    -   cv::CAP_PROP_WHITE_BALANCE_BLUE_U @copydoc cv::CAP_PROP_WHITE_BALANCE_BLUE_U
-    -   cv::CAP_PROP_WHITE_BALANCE_RED_V @copydoc cv::CAP_PROP_WHITE_BALANCE_RED_V
-    -   cv::CAP_PROP_RECTIFICATION @copydoc cv::CAP_PROP_RECTIFICATION
+    @param propId Property identifier from cv::VideoCaptureProperties (eg. cv::CAP_PROP_POS_MSEC, cv::CAP_PROP_POS_FRAMES, ...)
+    or one from @ref videoio_flags_others
     @param value Value of the property.
+    @return @c true if the property is supported by backend used by the VideoCapture instance.
+    @note Even if it returns @c true this doesn't ensure that the property
+    value has been accepted by the capture device. See note in VideoCapture::get()
      */
     CV_WRAP virtual bool set(int propId, double value);
 
     /** @brief Returns the specified VideoCapture property
 
-    @param propId Property identifier. It can be one of the following:
-    -   cv::CAP_PROP_POS_MSEC @copydoc cv::CAP_PROP_POS_MSEC
-    -   cv::CAP_PROP_POS_FRAMES @copydoc CAP_PROP_POS_FRAMES
-    -   cv::CAP_PROP_POS_AVI_RATIO @copydoc cv::CAP_PROP_POS_AVI_RATIO
-    -   cv::CAP_PROP_FRAME_WIDTH @copydoc cv::CAP_PROP_FRAME_WIDTH
-    -   cv::CAP_PROP_FRAME_HEIGHT @copydoc cv::CAP_PROP_FRAME_HEIGHT
-    -   cv::CAP_PROP_FPS @copydoc cv::CAP_PROP_FPS
-    -   cv::CAP_PROP_FOURCC @copydoc cv::CAP_PROP_FOURCC
-    -   cv::CAP_PROP_FRAME_COUNT @copydoc cv::CAP_PROP_FRAME_COUNT
-    -   cv::CAP_PROP_FORMAT @copydoc cv::CAP_PROP_FORMAT
-    -   cv::CAP_PROP_MODE @copydoc cv::CAP_PROP_MODE
-    -   cv::CAP_PROP_BRIGHTNESS @copydoc cv::CAP_PROP_BRIGHTNESS
-    -   cv::CAP_PROP_CONTRAST @copydoc cv::CAP_PROP_CONTRAST
-    -   cv::CAP_PROP_SATURATION @copydoc cv::CAP_PROP_SATURATION
-    -   cv::CAP_PROP_HUE @copydoc cv::CAP_PROP_HUE
-    -   cv::CAP_PROP_GAIN @copydoc cv::CAP_PROP_GAIN
-    -   cv::CAP_PROP_EXPOSURE @copydoc cv::CAP_PROP_EXPOSURE
-    -   cv::CAP_PROP_CONVERT_RGB @copydoc cv::CAP_PROP_CONVERT_RGB
-    -   cv::CAP_PROP_WHITE_BALANCE_BLUE_U @copydoc cv::CAP_PROP_WHITE_BALANCE_BLUE_U
-    -   cv::CAP_PROP_WHITE_BALANCE_RED_V @copydoc cv::CAP_PROP_WHITE_BALANCE_RED_V
-    -   cv::CAP_PROP_RECTIFICATION @copydoc cv::CAP_PROP_RECTIFICATION
-
-    @note When querying a property that is not supported by the backend used by the VideoCapture
-    class, value 0 is returned.
-     */
+    @param propId Property identifier from cv::VideoCaptureProperties (eg. cv::CAP_PROP_POS_MSEC, cv::CAP_PROP_POS_FRAMES, ...)
+    or one from @ref videoio_flags_others
+    @return Value for the specified property. Value 0 is returned when querying a property that is
+    not supported by the backend used by the VideoCapture instance.
+
+    @note Reading / writing properties involves many layers. Some unexpected result might happens
+    along this chain.<br>
+    <br>
```

Comment:
breaks help to keep lines under same _note_ paragraph and visual marker on the left. Solved with _code...endcode_